### DISCUSSION TITLE
draw.io or diagrams.net? Which one to use when?

### DISCUSSION THREAD

**Discussion by CrisRich58** [jgraph/drawio#2970] [General]
I recently came across draw.io and diagrams.net and would like to start using one of them on a Mac for drawing diagrams related to the use of energy and the environment.

But: which one should I use? draw.io or diagrams.net?

On this page from February 26, 2020 of your it is written: "Open source diagramming is moving to diagrams.net, slowly.” But since then, there are plenty of other messages on the blog on how to use draw.io. So, is it going to be discontinued or not? One can still download apps for both to use on Macs.

Is there a web page on which the two variants are compared, so one can better understand the use cases and when to choose which? I couldn’t find anything through Google.

**Comment by davidjgraph** [comment]
We use the two fairly interchangeable. The main problem was the security of draw.io domain itself. Changing the name of the desktop tool was too hard. See https://www.diagrams.net/integrations are the integrations we support.

**Reply by davidjgraph** [reply]
Don't reply on our behalf please. There is text at the bottom of the README that deals with use of the term "draw.io". The statement "it's safe to use the draw.io brand and namespace" is not correct.
